app/javascript/packs/core.tsx

```diff
@@ -1,10 +1,4 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-require('@rails/ujs').start()
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-require('@rails/activestorage').start()
```

Comment:
Can we remove these from package.json too?

Reply:
And remove the comments?